neurom/fst/tests/test_neuronfunc.py

```diff
@@ -134,7 +135,7 @@ class Mock(object):
 
 @nt.raises(Exception)
 def test_trunk_elevation_zero_norm_vector_raises():
-    _nf.trunk_origin_elevations(NRN)
+    _nf.trunk_origin_elevations(SWC_NRN)
```

Comment:
With the change of soma type and interpretation, the soma center has moved from [0,0,0] to [0.33, 0.1, 0].
Which causes, this test not to pass with the old neuron.